test/parser/base64.ics

```diff
@@ -1,13 +1,10 @@
 BEGIN:VCALENDAR
-ATTACH;FMTTYPE=text/plain;ENCODING=BASE64;VALUE=BINARY:TG9yZW
```

Comment:
Instead of changing the attachment value type to uri, I'd appreciate if you could keep it and then just add the parameter. Also, maybe you can clear out the real world data of that ics sample, both the real representation and the base64 encoded one, e.g.

```
ATTACH;FMTTYPE=text/plain;ENCODING=BASE64;VALUE=BINARY;X-BASE
 64-PARAM=UGFyYW1ldGVyCg==:WW91IHJlYWxseSBzcGVudCB0aGUgdGltZS
 B0byBiYXNlNjQgZGVjb2RlIHRoaXM/Cg==
```